torch/_functorch/vmap.py

```diff
@@ -391,18 +398,111 @@ def vmap(
         sequences out of the box.
     """
     _check_randomness_arg(randomness)
+    if not (chunk_size is None or chunk_size > 0):
+        raise ValueError("vmap: `chunk_size` should be greater than 0.")
```

Comment:
nit:
- it's slightly nicer if we say what the offending chunk_size is: f"vmap: chunk_size should be None or greater than 0. (got {chunk_size})".
- also we should mention chunk size can be None